tests/constructor/test_hydra_load.py

```diff
@@ -0,0 +1,24 @@
+import sys
+sys.path.append('../../')
+
+import hydra
+import omegaconf
+from omegaconf import OmegaConf, DictConfig
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import *
+
+from src.constructor.config_structure import ConfigParams, Phase
+
+
+@hydra.main(config_path='/workdir/rbayazitov/torchok/examples/configs/', config_name='example')
```

Comment:
Hard coded path.  
Overall, do you think we need this kind of test? Looks like you are testing Hydra's behaviour but not ours